lib/Runtime/ByteCode/OpCodes.h

```diff
@@ -828,6 +828,10 @@ MACRO_BACKEND_ONLY(     TrapIfZero,         Reg3,           OpSideEffect)
 MACRO_BACKEND_ONLY(     TrapIfUnalignedAccess, Reg3,        OpSideEffect)
 
 MACRO_EXTEND_WMS(       SpreadObjectLiteral, Reg2,          OpSideEffect|OpHasImplicitCall)
+MACRO_EXTEND_WMS(       StPropIdArrFromVar, Reg2Unsigned1,  OpSideEffect|OpHasImplicitCall)
+MACRO_EXTEND_WMS(       Restify,            Reg4,           OpSideEffect|OpHasImplicitCall)
+MACRO_EXTEND_WMS(       NewPropIdArr,       Reg1Unsigned1,  OpSideEffect|OpHasImplicitCall)
```

Comment:
I don't think `NewPropIdArr` has side effects or implicit calls, since it's only reading a compile-time constant. The others certainly do have options for side effects and implicit calls.